### PR TITLE
Make edx_django_service CONN_MAX_AGE default to 0.

### PR DESCRIPTION
**Pull request by mtyaka**
The default gunicorn worker class is 'gevent', which is not compatible with django's `CONN_MAX_AGE` because it leaks database connections.

We noticed this problem when our MySQL server stopped accepting new connections with 'too many open connections' error because there were hundreds of stale connections from ecommerce. Setting `CONN_MAX_AGE` to zero fixes the problem for us.

See: https://github.com/benoitc/gunicorn/issues/996#issuecomment-93301359

**Testing instructions**:

1. Spawn a new sandbox using the `edx_sandbox.yml` playbook.
2. Verify that services that are based on the `edx_django_service` role (ecommerce, discovery, analytics_api, credentials, ...) are configured with `CONN_MAX_AGE` value `0`. The configurations can be found in `/edx/etc/<service-name>.yml` file.

**Author notes and concerns**:

This changes the default for `CONN_MAX_AGE` from `60` to `0`, which is not backwards compatible. Since the default configuration of `60` was buggy, I think that's acceptable, but I'm also fine with reverting the default back to `60` as long as the value is overridable, if that's preferred.

**Reviewers**:
- [x] @kaizoku
- [ ] edX

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mtyaka! I've created [OSPR-2502](https://openedx.atlassian.net/browse/OSPR-2502) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by mduboseedx** [comment]
@edx/devops Could you give this a review?

**Comment by kaizoku** [comment]
This should fix our issues. Thanks for doing the research on this one @mtyaka 
:+1:

**Comment by mtyaka** [comment]
@mduboseedx Any idea when this could get reviewed?

**Comment by mduboseedx** [comment]
@mtyaka I'm checking in with our DevOps team. I'll let you know when I have more info.

**Comment by mtyaka** [comment]
Thank you @mduboseedx  :)

**Comment by kaizoku** [comment]
@mduboseedx has there been any update on this one?
Just checking as it's been a bit, and this could be a problem for other providers with many users.

**Comment by mduboseedx** [comment]
@feanil Could you or someone on your team give this a look?

**Comment by UmanShahzad** [comment]
This is a pretty simple config change that effectively fixes some buggy behavior.

File: playbooks/roles/edx_django_service/defaults/main.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@ edx_django_service_caches:
 edx_django_service_default_db_host: 'localhost'
 edx_django_service_default_db_name: '{{ edx_django_service_name }}'
 edx_django_service_default_db_atomic_requests: false
+edx_django_service_default_db_conn_max_age: 0
```
Comment (feanil) [inline playbooks/roles/edx_django_service/defaults/main.yml]: By changing this default here, we are changing the behavior of all  services already running.  I think making this configurable but then over-ridable per service makes sense but the default should remain 60 for now I think.

**Review by feanil — CHANGES_REQUESTED** [review]

File: playbooks/roles/edx_django_service/defaults/main.yml (hunk 1 above)
Comment (UmanShahzad) [reply playbooks/roles/edx_django_service/defaults/main.yml]: Thanks, good point. cc @kaizoku since this recently bit us, can you check this out?

Comment (kaizoku) [reply playbooks/roles/edx_django_service/defaults/main.yml]: Alright, I think we can leave the default at 60, but use the extra variable to set this to 0 on our instances.

**Comment by natabene** [comment]
@kaizoku  Please let me know once it is ready to be looked at again. Thanks

**Comment by kaizoku** [comment]
@natabene, I've updated the default to match the original 60 seconds again.

**Comment by natabene** [comment]
@feanil  What do you think?

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@jdmulloy can you do a second review of this when you have a sec?

**Review by jdmulloy — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@mtyaka 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/4666&entry.1484655318=https://github.com/mtyaka&entry.752974735=2018-07-03+15:54&entry.1917517419=2018-11-14+21:44&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by kaizoku** [comment]
Thanks for merging this @jdmulloy !

**Pull request merged**